proposals/stage-0/code-of-conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -0,0 +1,71 @@
+# Foundation Code of Conduct Requirements
+
+It is important that there be a Code of Conduct (CoC) which
+is documented and enforced for the _$FOUNDATION_ and the
+projects within it. This document defines the requirements
+for a CoC for the _$FOUNDATION_ and the projects which are part
+of the foundation.
+
+
+## Code of Conduct
+
+The _$FOUNDATION_ has adopted the
+[Contributor Covenant v1.4.1](https://www.contributor-covenant.org/version/1/4/code-of-conduct.md)
+as its code of conduct and it applies to all activities/spaces at the foundation level. In
```

Comment:
What does "at the foundation level" mean here?

Reply:
(Again, a question/comment that does not *require* immediate attention. Feel free to merge without addressing it at this time.)

Reply:
At the foundation level means that repos, meetings etc which are under the Foundation but not in a specific project.  A good example are CPC meetings or repos which are in a github organization setup for the Foundation as opposed to a project.

Reply:
Would it be simpler and clearer to say "it applies to all foundation activities and spaces" rather than "it applies to all activities/spaces at the foundation level"?

Reply:
(^^^^ Legitimately asking. I don't know if that somehow changes the meaning in a subtle way. I would expect that it might make the applicability *more* expansive, but I would also imagine that would be desirable.)

Reply:
I'm happy with your suggested wording, will pull in.